jekyll/_cci2/artifacts.md

```diff
@@ -142,9 +142,9 @@ curl https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/:build_n
 <artifacts.txt xargs -P4 -I % wget %?circle-token=$CIRCLE_TOKEN
 ```
 
-Note 1: Replace all the variables above that start with a : with real values for your project (don't include the colon).
+Note 1: Replace all the variables above that start with a *:* (colon) with real values for your project (don't include the colon).
```

Comment:
I'd swap the symbol with the word for the parentheses:
"Note 1: Using the code above, replace variables starting with a colon with real values for your project."